tests/test_tracking.py

```diff
@@ -45,18 +47,29 @@ def _get_port():
                 continue
         else:
             return port
+
+
 PORT = _get_port()
 
 
 class SampleTracker(TrackerBase):
     def record(self, header, exception):
         db = dbm.open(db_file, 'w')
+        key = "%s:%s" % (header.request_id, header.seq)
+        db[key.encode("ascii")] = pickle.dumps(header.__dict__)
+        db.close()
+
 
+class OldTracker(old_TrackerBase):
```

Comment:
Different trackers of different versions should also be imported from the module of that version.